probe_scraper/probe_expiry_alert.py

```diff
@@ -189,7 +206,8 @@ def check_bugzilla_user_exists(email: str, api_key: str):
     user_response = requests.get(BUGZILLA_USER_URL + "?match=" + email,
                                  headers=bugzilla_request_header(api_key))
     user_response.raise_for_status()
-    return len(user_response.json()["users"]) > 0
+    users = user_response.json()["users"]
+    return len(users) > 0 and users[0]["can_login"]
```

Comment:
What is `users[1]` can login?

Reply:
`can_login` tells us if the account is disabled or not

Reply:
Right, but what if e.g. the second account is not disabled?

Reply:
Oh I see what you mean.  Since this uses the `match` query param this can include multiple users like `abc@a.com` and `abc@a.com2`.  What should be done is to use `name` so that `len(users)` can only be 1 or 0.  We can raise an exception if not 0 or 1 to ensure this